src/app/theme/toolbar.css

```diff
@@ -135,6 +135,20 @@
 		& button[aria-label="Add a task list"] svg {
 			margin-left: -3px;
 		}
+
+		& .ck-insert-table-dropdown-grid-box {
+			margin: var(--ck-insert-table-dropdown-box-margin);
+			border: 1px solid var(--ck-color-base-border) !important;
+
+			& .ck-button.ck-on,
+			& .ck-button:not(.ck-disabled):hover {
+				border-color: var(--ck-color-focus-border);
+				background: var(--ck-color-focus-outer-shadow);
+			}
+		}
+		& .ck-insert-table-dropdown__label {
+			color: inherit;
```

Comment:
I think this rule does nothing at the moment. If I missed something please just give an explanation what it is used for.